File: LeetCode/LeetCode Practice Problems/1748- Sum of Unique Elements.cpp
class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        map<int,int> m;
        for(auto i : nums) {
            m[i]++;
        }
        
        int s = 0;
        for(auto i:m) {
            if(i.second == 1) {
                s += i.first;
            }
        }
        
        return s;
    }
};

File: LeetCode/LeetCode Practice Problems/1748-Sum of Unique Elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        int freq[] = new int[110];
        
        for(int num: nums) {
            freq[num]++;
        }
        
        int ans = 0;
        for(int i=0;i<=100;i++) {
            ans += (freq[i] == 1)?i:0;
        }
        
        return ans;
    }
}
